lib/filesystem.nix

```diff
@@ -18,7 +18,10 @@ let
     ;
 
   inherit (lib.filesystem)
+    pathIsDirectory
+    pathIsRegularFile
     pathType
+    packagesFromDirectoryRecursive
```

Comment:
Is this line necessary?  I *think* `packagesFromDirectoryRecursive` is already in scope